cddl/measurement-values-map.cddl

```diff
@@ -13,5 +13,6 @@ measurement-values-map = non-empty<{
   ? &(ueid: 9) => ueid-type
   ? &(uuid: 10) => uuid-type
   ? &(name: 11) => text
+  ? &(cryptokeys: 12) => [ + $crypto-key-type-choice ]
```

Comment:
This is an array because there can be multiple keys that are protected by the environment. The verifier is checking that the same list of keys reported in Evidence is included in Reference Values. The Verifier will not use or verify the keys unless the RP or a downstream verifier instructs this. But we can say the same for any measurement (that RP or downstream verifier may post process the claims). The default Verifier action is to compare each Evidence key at an array position with each Reference Value key at the same array position.